kivy/uix/label.py

```diff
@@ -16,12 +16,50 @@
     # size
     l = Label(text='Hello world', font_size='20sp')
 
+.. _kivy-uix-label-sizing-and-text-content:
+
+Sizing and text content
+---------------------------
+
+By default, the size of :class:`Label` is not affected by :attr:`~Label.text`
+content and the text is not affected by the size. In order to control
+sizing, you must specify :attr:`~Label.text_size` to constrain the text
+and/or bind :attr:`~Label.size` to :attr:`~Label.texture_size` to grow with
+the text.
+
+For example, this label's size will be set to the text content
+(plus :attr:`~Label.padding`):
+
+.. code-block:: python
+
+    Label:
+        size: self.texture_size
+
+This label's text will be clipped to the specified width and height:
```

Comment:
I think I want to change this to:
> This label's text will wrap at the specified width and be clipped to the height:

Alternatively, is it better style to recommend this?:
```python
Label:
    size: cm(6), cm(4)
    text_size: self.size
```